cli/args/flags.rs

```diff
@@ -1217,11 +1227,14 @@ Ignore formatting a file by adding an ignore comment at the top of the file:
         .help("Define how prose should be wrapped. Defaults to always."),
     )
     .arg(
-      Arg::new("options-semi")
-        .long("options-semi")
+      Arg::new("options-no-semicolons")
+        .long("options-no-semicolons")
+        .min_values(0)
+        .max_values(1)
         .takes_value(true)
-        .possible_values(["prefer", "asi"])
-        .help("Use semi colons. Defaults to prefer."),
+        .require_equals(true)
+        .possible_values(["true", "false"])
+        .help("Don't use semi colons except where necessary."),
```

Comment:
Should probably be "semicolon", at least according to Wikipedia, Merriam-Webster and Cambridge Dictionary.

Reply:
Thanks! Yes it should